beem/cmds/publish.py

```diff
@@ -207,8 +210,13 @@ def run(options):
         agg_stats = beem.aggregate_publish_stats(stats_set)
         agg_stats["time_total"] = time_end - time_start
         beem.print_publish_stats(agg_stats)
+        if options.json is not None:
+            beem.json_dump_stats(agg_stats, options.json)
     else:
         agg_stats_set = [beem.aggregate_publish_stats(x) for x in stats_set]
         for x in agg_stats_set:
             x["time_total"] = time_end - time_start
         [beem.print_publish_stats(x) for x in agg_stats_set]
+        if options.json is not None:
+            [beem.json_dump_stats(x, options.json) for x in agg_stats_set]
```

Comment:
this is only going to write the last file, it will just keep truncating it each time.  Is that really what you meant?